web_viewer/index.html

```diff
@@ -328,8 +328,12 @@
 
             check_for_panic();
 
-            let url = determine_url();
-            handle.start("the_canvas_id", url).then(on_app_started).catch(on_wasm_error);
+            let url = new URLSearchParams(window.location.search).get("url");
+            let fallback_to_websockets = false; // We're on `app.rerun.io`, so don't fallback to WebSockets.
```

Comment:
I believe `web_viewer/index.html` is used in more places than just `app.rerun.io`, including just running `cargo rerun-web` and when calling `serve` from our logging SDK, e.g. when running `cargo r -p raw_mesh -- --serve`

Reply:
When I run `cargo rerun-web` or `cargo r -p raw_mesh -- --serve`, it sends me to `http://localhost:9090/?url=ws://localhost:9877`, which works as expected.